### PR TITLE
2 September

### PR DESCRIPTION
**Pull request by aadi163**

### CODE REVIEW

File: app/controllers/order_items_controller.rb
Comment (ShahirMansuri786) [inline app/controllers/order_items_controller.rb:1]: Please check your email delivering when your email is delivering so only one product details will deliverd to customer

File: app/controllers/wishlist_items_controller.rb [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-  before_action :find_product  , only: [:add_to_wishlist]
-  before_action :authenticate_user! , only: [:add_to_wishlist]
-
+  before_action :authenticate_user! 
 
   def add_to_wishlist
     wishlist = current_user.wishlist || current_user.create_wishlist
     wishlist_item = wishlist.wishlist_items.find_or_initialize_by(product: @product)
-
+    @product = Product.find(params[:id])
```
Comment (ShahirMansuri786) [inline app/controllers/wishlist_items_controller.rb]: Please write this line starting of your action and if you are not using in another file so please declare as local variable

File: app/controllers/order_items_controller.rb
Comment (aadi163) [reply app/controllers/order_items_controller.rb:1]: I was just checking that product details are showing or not in the mail. but now I will correct this problem

File: app/controllers/wishlist_items_controller.rb (hunk 1 above)
Comment (aadi163) [reply app/controllers/wishlist_items_controller.rb]: I have changed this variable and arranged the code .

File: app/controllers/order_items_controller.rb [hunk 2]
```diff
@@ -10,7 +11,7 @@ def final_order
       cart = Cart.where(user_id: current_user.id)
       @cart_items = Cartdataitem.where(cart_id: cart)
     end
-    @useraddresses = Useraddress.all
+    @useraddresses = Useraddress.where(user_id: current_user)
```
Comment (vinayaklovanshi) [inline app/controllers/order_items_controller.rb]: Why you are passing current_user in user_id?
Current_user is object here.

Wrong variable name

**Review by vinayaklovanshi — CHANGES_REQUESTED** [review]

File: app/controllers/order_items_controller.rb (hunk 2 above)
Comment (aadi163) [reply app/controllers/order_items_controller.rb]: sure sir.  I will correct this.

**Pull request merged**